dali/pipeline/operator/op_schema.cc

```diff
@@ -189,6 +189,7 @@ std::string OpSchema::GetArgumentDefaultValueString(const std::string &name) con
 
   auto &val = *value_ptr;
   auto str = val.ToString();
+
```

Comment:
revert? Otherwise this file would should changes from this PR, when there are actually none